transformer/dockerfilegenerator/java/gradleanalyser.go

```diff
@@ -426,11 +423,10 @@ func (t *GradleAnalyser) Transform(newArtifacts []transformertypes.Artifact, alr
 				Type: artifacts.EarArtifactType,
 				Configs: map[transformertypes.ConfigType]interface{}{
 					artifacts.EarConfigType: artifacts.EarArtifactConfig{
-						DeploymentFile:                    archiveName,
-						JavaVersion:                       javaVersion,
-						BuildContainerName:                common.DefaultBuildContainerName,
-						DeploymentFileDirInBuildContainer: filepath.Join(t.GradleConfig.AppPathInBuildContainer, buildPath, destinationPath),
-						EnvVariables:                      envVariablesMap,
+						DeploymentFilePath: archiveName,
```

Comment:
Shouldn't it be filepath.Join(t.GradleConfig.AppPathInBuildContainer, buildPath, destinationPath). Also, we need DeploymentFileName.